test/integration/components.cc

```diff
@@ -1482,123 +1684,3 @@ TEST_F(ComponentsTest, World)
   components::World comp3;
   comp3.Deserialize(istr);
 }
-
-/////////////////////////////////////////////////
-TEST_F(ComponentsTest, Scene)
```

Comment:
Looks like a bad merge? I see the tests `ParticleEmitter` and `ParticleEmitterCmd` in the new file, but `Scene` is missing.

Reply:
I was re-ordering the tests so that they adhered to alphabetical order. The `Scene` test still shows up in the file when I take a look: https://github.com/ignitionrobotics/ign-gazebo/blob/76cfcf4b4c25919b28278ee4b7d25d217c13bf97/test/integration/components.cc#L1493

If you'd like, I can undo the ordering changes for these tests and leave them as-is. I was just trying to make the ordering more consistent.

Reply:
Ah, not sure how I missed it. Thanks.